conf/messages.en

```diff
@@ -131,6 +131,9 @@ amountDue.bands.h3 = ECL fee to pay
 amountDue.bands.p1 = There are four size bands. These are based on the amount of UK revenue you earned in your relevant accounting period.
 amountDue.bands.p2 = The levy is paid as a fixed fee if your band size is medium or higher.
 amountDue.bands.p3 = The fee you need to pay is reduced if you did not carry out AML-regulated activity for the full financial year.
+amountDue.bands.nil.p1 = You do not need to pay the Economic Crime Levy (ECL) for the 20XX-20XX financial year.
```

Comment:
I think the years shown are meant to be dynamic based on the period the return is for.

Reply:
Tony is right - these years need to come from the obligation stored in the return data.... 20XX makes no sense 😄